code/modules/projectiles/guns/ballistic/rifle.dm

```diff
@@ -53,6 +64,10 @@
 	desc = "This piece of junk looks like something that could have been used 700 years ago. It feels slightly moist."
 	icon_state = "moistnugget"
 	item_state = "moistnugget"
+	can_sawoff = TRUE
+	sawn_name = "\improper Mosin Obrez"
+	sawn_desc = "Огонь!"
```

Comment:
This needs to be in english

Reply:
Would you prefer "Mosin Obrez" to be changed to the standard "sawn-off Mosin" as well, or is that fine staying as is?

Reply:
Not against that as much since it uses english characters

Reply:
It's an English loan word that just refers to a sawn-off firearm. Colloquially it refers *explicitly* to a sawn-off Mosin
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/97719613/82870ae6-9f87-4315-b255-0054a8ff6274)

Reply:
Marking this as resolved, as Obrez *is* an English word, although a loanword.